templates/zerver/help/sidebar.md

```diff
@@ -35,6 +35,7 @@
 * [Reply to a message](/help/reply-to-a-message)
 
 ## Reading Messages
+* [How unread messsage counts and navigation work](/help/unread-counter-and-pointer)
```

Comment:
Good idea to put this at the top of the section.  Maybe we could shorten the title a little bit, e.g. "Navigation and unread counts"?